api/location/schema.py

```diff
@@ -0,0 +1,39 @@
+import graphene
+from graphene import relay,Schema
+from graphene_sqlalchemy import (
+    SQLAlchemyObjectType, 
+    SQLAlchemyConnectionField
+)
+
+from api.location.models import Location as LocationModel
+
+from api.floor.models import Floor as FloorModel 
+from api.room.models import Room as RoomModel
+
+from api.equipment.models import Equipment 
+
+class Location(SQLAlchemyObjectType):
+    class Meta:
+        model = LocationModel
+
+class Query(graphene.ObjectType):
+    node = relay.Node.Field()
+    all_locations = graphene.List(Location)
+    get_rooms_in_a_location =graphene.List(
```

Comment:
I think this should be **get_location** not **get_rooms_in_a_location**

Reply:
According to the flow on the front end, one needs to query rooms in a hierarchy. So the rooms_in_a_location will be from the location, the block, the floor and finally rooms. Please refer to the screenshot.

Reply:
Thanks for that insight. However, according to the story requirements and the how the flow of the application goes on the front end, these room queries are meant to be hierarchical. That is why I went for this naming convention. Kindly take a look at the story on the PT board for more clarity.